dvc/command/get.py

```diff
@@ -10,7 +10,28 @@
 
 
 class CmdGet(CmdBaseNoRepo):
+    def _run_get_url(self):
+        from dvc.api import get_url
+
+        try:
+            url = get_url(
+                self.args.path, repo=self.args.url, rev=self.args.rev
+            )
```

Comment:
Btw, another discrepancy is that `get_url` doesn't work with git files 🙁 Need to at least show a proper error message for now. We could consider showing a url to a raw file though, e.g. kinda like raw.github.com/... , but i'm not sure if that is natively supported by git somehow, especially if it is not github. 🤔

Reply:
Handled by catching exception for `NotDvcRepoError`. For now, we won't support git-only repo files. We could theoretically generate a URL for GitHub for convenience, but, I'd leave it as an enhancement for now.